tests/PHPStan/Analyser/data/pow.php

```diff
@@ -177,3 +177,23 @@ function doFoo(int $intA, int $intB, string $s, bool $bool, $numericS, float $fl
 	assertType('*ERROR*', $bool ** $arr);
 	assertType('*ERROR*', $bool ** []);
 };
+
+function invalidConstantOperands(): void {
+	assertType('*ERROR*', 'a' ** 1);
```

Comment:
php fatal errors on regular (non-numeric strings). I think we should too

https://3v4l.org/iTXRN